x-pack/plugins/apm/public/tutorial/config_agent/commands/java.ts

```diff
@@ -5,10 +5,17 @@
  * 2.0.
  */
 
+export const javaVariables = {
+  apmServiceName: 'Delastic.apm.service_name',
+  secretToken: 'Delastic.apm.secret_token',
+  apmServerUrl: 'Delastic.apm.server_urls',
+  apmEnvironment: 'Delastic.apm.environment',
+};
+
 export const java = `java -javaagent:/path/to/elastic-apm-agent-<version>.jar \\
--Delastic.apm.service_name=my-application \\
--Delastic.apm.server_urls={{{apmServerUrl}}} \\
```

Comment:
Is this going to be replaced with something else?

Reply:
@gbamparop what do you mean?

Reply:
I was referring to `todo check java agent. it also has ServerUrls`

Reply:
I'm verifying it with java agent team. Most likely it will get updated to use singular `Delastic.apm.server_urls`

Reply:
verify it and updated here: 3b90a58276f7bd792faf6f8dc0659636ec0a31df

Reply:
Are you going to add it back in here (`x-pack/plugins/apm/public/tutorial/config_agent/commands/java.ts`) as well?

Reply:
Yes I updated in java commands if that's what you mean

- https://github.com/elastic/kibana/blob/3b90a58276f7bd792faf6f8dc0659636ec0a31df/x-pack/plugins/apm/public/tutorial/config_agent/commands/java.ts#L11

Reply:
nice one thanks!